clang/test/Driver/integrated-as.c

```diff
@@ -1,4 +1,4 @@
-// XFAIL: target={{.*}}-aix{{.*}}
+// XFAIL: target={{.*}}-aix{{.*}}, target=riscv{{.*}}
```

Comment:
This causes different behaviors with the default target triple, which is probably not perfect.

I think we can change the -mrelax-all check with an explicit target triple and then add a `--target=riscv64`.

Reply:
What triple should i use for the -mrelax-all test?

Reply:
Perhaps the commonplace `--target=x86_64`